utilities/src/main/java/org/craftercms/commons/config/profiles/aws/S3ProfileMapper.java

```diff
@@ -31,13 +31,21 @@
 public class S3ProfileMapper extends AbstractAwsProfileMapper<S3Profile> {
 
     private static final String CONFIG_KEY_BUCKET = "bucketName";
+    private static final String CONFIG_KEY_ENDPOINT = "endpoint";
 
     @Override
     protected S3Profile mapProfile(HierarchicalConfiguration<ImmutableNode> profileConfig)
             throws ConfigurationException {
         S3Profile profile = super.mapProfile(profileConfig);
         profile.setBucketName(getRequiredStringProperty(profileConfig, CONFIG_KEY_BUCKET));
 
+        /**
+         * Check if <endpoint> is provided as it is optional
+         */
+        if(profileConfig.getString(CONFIG_KEY_ENDPOINT) != null && !profileConfig.getString(CONFIG_KEY_ENDPOINT).isEmpty()){
```

Comment:
Since this property is not required, please use `ConfigUtils.getStringProperty`, no need for the `if`.

Reply:
Hi Alfonso,
So remove the if statement and replace with below?

`profile.setEndpoint(getStringProperty(CONFIG_KEY_ENDPOINT));`

Reply:
Correct, but you will also need to import the method statically like it's done with `getRequiredStringProperty`.

Reply:
Thanks!